pkg/controller/syncmachineset/gcpactuator.go

```diff
@@ -0,0 +1,126 @@
+package syncmachineset
+
```

Comment:
i tried to move this and ended up with an import loop of pkg/controller/syncmachineset importing pkg/controller/syncmachineset/gcp, and pkg/conroller/syncmachineset/gcp importing pkg/controller/syncmachineset (so that we can do the Actuator = &GCPActuator{} compile-time check)

Reply:
actually, nevermind. i just needed to separate out the actuator definition into its own file. updated PR to move the gcp actuator into its own directory/package.

Reply:
We have generally not taken the approach of separating into directories by platform. We separate into directories by function instead. I would rather see the gcp actuator live in the same directory as all of the other actuators.